kafka/client_async.py

```diff
@@ -526,6 +526,8 @@ def poll(self, timeout_ms=None, future=None, delayed_tasks=True):
             timeout_ms = 100
         elif timeout_ms is None:
             timeout_ms = self.config['request_timeout_ms']
+        elif not isinstance(timeout_ms, int) and not isinstance(timeout_ms, float):
```

Comment:
`isinstance()` supports checking against multiple types in one call:

    isinstance(timeout_ms, (int, float))

Reply:
Nice. I'll change it.